ccpp/config/ccpp_prebuild_config.py

```diff
@@ -184,7 +187,10 @@
     'FV3/ccpp/physics/physics/gcm_shoc.F90'                      : [ 'slow_physics' ],
     'FV3/ccpp/physics/physics/get_prs_fv3.F90'                   : [ 'slow_physics' ],
     'FV3/ccpp/physics/physics/gfdl_cloud_microphys.F90'          : [ 'slow_physics' ],
-    'FV3/ccpp/physics/physics/gfdl_fv_sat_adj.F90'               : [ 'fast_physics' ],
+    'FV3/ccpp/physics/physics/gfdl_fv_sat_adj.F90'               : [ 'slow_physics' ],
```

Comment:
is this correct?  gfdl_fv_sat_adj.F90 should remain in "fast_physics", right?

Reply:
Yes, I will revert this.